tests/e2e/load_test.ml

```diff
@@ -0,0 +1,143 @@
+open Cmdliner
+open Crypto
+open Node
+open Helpers
+
+type wallet = {
+  key_hash : Key_hash.t;
+  secret : Secret.t;
+}
+
+(* The wallets are hard-coded to make it easier to deposit the initial ticket
+   and to enable expect tests on the output in the future. *)
+let alice_wallet =
+  {
+    key_hash =
+      Key_hash.of_string "tz1RPNjHPWuM8ryS5LDttkHdM321t85dSqaf" |> Option.get;
+    secret =
+      Secret.of_string "edsk36FhrZwFVKpkdmouNmcwkAJ9XgSnE5TFHA7MqnmZ93iczDhQLK"
+      |> Option.get;
+  }
+let bob_wallet =
+  {
+    key_hash =
+      Key_hash.of_string "tz1h1oFuYsCorjxekQ59bUe1uDGhuYvEx9ob" |> Option.get;
+    secret =
+      Secret.of_string "edsk326F1xfCvHFw1LWhgtrwcm6DnFoHCmjjWX4vcWsJCbqmujJQVs"
+      |> Option.get;
+  }
+
+let validators_uris =
+  ["http://localhost:4440"; "http://localhost:4441"; "http://localhost:4442"]
+
+let get_random_validator_uri () =
+  List.nth validators_uris (Stdlib.Random.int 3) |> Uri.of_string
+
+let get_current_block_level () =
+  let validator_uri = get_random_validator_uri () in
+  let block_level =
+    Lwt_main.run @@ Networking.request_block_level () validator_uri in
+  block_level.level
+
+(* Assumes that the bytes of the ticket are empty. This simplifies things
+   quite a bit, since we don't have to query the contents of the ticket
+   or serialize and then parse the bytes *)
+let make_ticket ticketer =
+  let contract = Tezos.Contract_hash.of_string ticketer |> Option.get in
+  let ticketer = Tezos.Address.Originated { contract; entrypoint = None } in
+  Core.Ticket_id.{ ticketer; data = Bytes.empty }
+
+let nonce = ref 0l
+
+let do_transaction ~validator_uri ~block_level ~ticket ~sender ~recipient
+    ~amount =
+  nonce := Int32.add 1l !nonce;
+  let amount = Core.Amount.of_int amount in
+  let transaction =
+    Protocol.Operation.Core_user.sign ~secret:sender.secret ~nonce:!nonce
+      ~block_height:block_level
+      ~data:
+        (Core.User_operation.make
+           ~sender:(Core.Address.of_key_hash sender.key_hash)
+           (Transaction { destination = recipient.key_hash; amount; ticket }))
+  in
+  Lwt.async (fun () ->
+      Networking.request_user_operation_gossip
+        { user_operation = transaction }
+        validator_uri);
+  transaction
+
+let spam_transactions ~ticketer n () =
+  (* Hard-coded for now. TODO: get these dynamically, see https://github.com/marigold-dev/deku/pull/450 *)
+  let validators_uris =
```

Comment:
You can call the function `validators_uris`, it is the same for `get_random_validator_uri`

Reply:
Addressed :thumbsup: